.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,770 @@
+
+   - **A title**. The PR title must describe the changes you made. For example: `Add getting started page to API documentation`.
+   - **A description**. A clear description can help PR reviewers understand the changes you made in your PR. It's always good to walk through the process of how a reviewer can test your changes.
+   - **A related issue**. [Link the issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) that you worked on and add a keyword of 'Closes', 'Fixes', or 'Resolves' in front of it. For example, `Closes #123`, `Fixes #234`, etc. You can find the issue number right next to the issue's title. Linking the issue number automatically closes it once your PR is merged.
+
+3. Provide screenshots for visual changes if necessary.
+
+> [!TIP]
+>
+> Refer to [PR #369](https://github.com/mautic/mautic-community-handbook/pull/369) in the Mautic Community Handbook for an example of a well-documented PR.
+
+### After submitting a PR
+
+1. Ensure that all checks pass. If the linting build or prose fails, debug and fix it until all pass. If you have questions or need help, feel free to tag the `@mautic/education-team-leaders` in the comment.
+2. Please don't DM maintainers on Slack to review or ask feedback and questions about your PR.
+
+   If you'd like feedback or ask questions about your PR, tag `@mautic/education-team-leaders` in the comment of your PR or use the `#t-education` channel on Slack. That way, not only maintainers, but the community can help you get unstuck. The team always receives a notification whenever an incoming PR is submitted. If you haven't received a review within a week, you can tag them in the PR comments to ask for an estimated review time.
+
+3. Keep your branch up to date while waiting for review.
+4. Respond and address the reviewer's feedback. Please don't request a review until you've addressed all feedback.
+
+## Contributing workflow
+
+In this section, you can find the contributing workflow and best practices for contributing to this project.
+
+### Forking the repository
+
+Forking this repository is the first step you need to take before proceeding. Forking means making a copy of a repository to your GitHub account.
+
+To fork this repository:
+
+1. In the main page of [mautic/developer-documentation-new](https://github.com/mautic/developer-documentation-new), click the 'Fork' button at the top.
+
+   ![Fork button on GitHub](../assets/images/fork_button_github.png)
+
+2. Choose your username in the 'Owner *' dropdown menu. **Don't select an organization here. Always choose your personal account**; otherwise, maintainers can't collaborate or fix issues in your PR.
+
+   ![Choose fork owner on GitHub](../assets/images/choose_fork_owner_github.png)
+
+3. Uncheck the option of 'Copy the DEFAULT-BRANCH-NAME branch only' because later on, you want to be able to clone multiple branches.
```

Comment:
```suggestion
3. Remove the check in the option of 'Copy the DEFAULT-BRANCH-NAME branch only' because later on, you want to be able to clone multiple branches.
```